app/models/endpoint.rb

```diff
@@ -13,6 +13,12 @@ def verify_ssl?
     verify_ssl != OpenSSL::SSL::VERIFY_NONE
   end
 
+  def self.default_database_name_for(type)
```

Comment:
I really was not sure about this. Maybe I should make a class "EndpointConfiguration"? But it will have only this Method, and if so where should I put it?